.changeset/serious-tigers-clap.md

```diff
@@ -0,0 +1,5 @@
+---
+"@directus/sdk": patch
+---
+
+Fixed limit=-1 support in the SDK
```

Comment:
Just curious, should we make this less technical for the notes? Something like

```suggestion
Fixed support for unlimited query results in the SDK
```

Reply:
Yeah, good idea 👍 
My suggestion:
```suggestion
Fixed support for using limit parameter with unlimited (-1) value in the SDK
```